jobs/DetermineBranchingFactor.py

```diff
@@ -1,22 +1,19 @@
-import WikipediaApi
+from src import WikipediaApi
 
 
-def determine_branching_factor(samples=1000):
+def determine_branching_factor(samples):
     """
     Determine the branching factor of Wikipedia articles through random sampling
     :param samples: The number of random articles to sample
     :return: The average number of links per article sampled
     """
-    wikiApi = WikipediaApi.WikipediaApi()
+    wiki_api = WikipediaApi.WikipediaApi()
```

Comment:
It might be better to make a `JobRunner` script or something, since `WikiRacer` just runs the search with a heuristic, so we shouldn't add a ton of job options to it. We can add it in after both our PRs are merged.